### ISSUE TITLE
Social icon widget shows white icons after latest update

### ISSUE THREAD

**Issue by jormeer** [elementor/elementor#12900]
## Prerequisites

- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x] The issue still exists against the latest stable version of Elementor.

## Isolating the problem

- [x] This bug happens with only Elementor plugin active (and Elementor Pro).
- [x] This bug happens with a default WordPress theme active.
- [x] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
```
<!-- For bug reports, let us know about your system environment: Copy and paste the system info report from Elementor => System info, and paste it here or in http://pastebin.com/ -->
```
</details>

**Comment by jormeer** [comment]
Social icon widget shows white icons after latest update. Have this on all my sites.

**Comment by jormeer** [comment]
I know how it happens. When having a font size in global site settings for Links it affects the icons size. I have it on 1.05 em and it causes to shrink the icon.
with 1,05 em. See screenshots

 https://ibb.co/4S4G37d

with px size 

https://ibb.co/hy926sr

The problem is that the icons are linked to global link font-size ?

**Comment by brahimvandenbrande** [comment]
I added custom css to fix the issue : 

`.elementor-social-icon {
    font: icon;
}`

**Comment by shilo-ey** [comment]
This issue has been resolved in Elementor v3.1.2

Feel free to update

Thanks!
